client/landing/jetpack-cloud/sections/scan/main.tsx

```diff
@@ -112,8 +110,11 @@ class ScanPage extends Component< Props > {
 	}
 
 	renderScanOkay() {
-		const { siteID, siteSlug, moment, lastScanTimestamp, dispatchRecordTracksEvent } = this.props;
+		const { siteID, siteSlug, moment, scanState, dispatchRecordTracksEvent } = this.props;
 
+		const lastScanTimestamp = scanState?.mostRecent?.timestamp
+			? Date.parse( scanState?.mostRecent?.timestamp )
```

Comment:
Typescript complaints about this line because `scanState.mostRecent.timestamp` is Date object and `Date.parse` expects a string as argument. We can use `getTime()` and obtain the same result.
```js
? scanState?.mostRecent?.timestamp.getTime()
```